src/main/java/nl/novi/techiteasy1121/models/Television.java

```diff
@@ -0,0 +1,171 @@
+package nl.novi.techiteasy1121.models;
+import jakarta.persistence.*;
+
+@Entity
+@Table (name="Televisions")
+
+public class Television {
+@GeneratedValue (strategy = GenerationType.IDENTITY)
+@Id
+
+private Long id;
+
+private String type;
+
+private String name;
+
+private String brand;
+
+private double price;
+
+private double availableSize;
+
+    private Integer refreshRate;
+    private String screenType;
+    private String screenQuality;
+    private Boolean smartTv;
+    private Boolean wifi;
+    private Boolean voiceControl;
+    private Boolean hdr;
+    private Boolean bluetooth;
+    private Boolean ambiLight;
+    private Integer originalStock;
+    private Integer sold;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
```

Comment:
Deze setter is niet heel netjes, we willen natuurlijk niet een id aanpassen waardoor we de database id generatie overhoop halen.